From 90d47c8a95042b4b499c93c4459a0ae0b1408b6b Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Mon, 8 Apr 2024 16:33:52 +0200
Subject: [PATCH] com.webos.service.memorymanager: Fix outbound permission
 errors

Fixes:

Apr 08 16:02:40 qemux86-64 ls-hubd[96]: [] [pmlog] ls-hubd LSHUB_NO_OUT_PERMS {"DEST_APP_ID":"com.webos.applicationManager","SRC_APP_ID":"com.webos.service.memorymanager","EXE":"/usr/sbin/memorymanager","PID":728} "com.webos.service.memorymanager" does not have sufficient outbound permissions to communicate with "com.webos.applicationManager" (cmdline: /usr/sbin/memorymanager)
Apr 08 16:02:40 qemux86-64 ls-hubd[96]: [] [pmlog] ls-hubd LSHUB_NO_OUT_PERMS {"DEST_APP_ID":"com.palm.webappmanager","SRC_APP_ID":"com.webos.service.memorymanager","EXE":"/usr/sbin/memorymanager","PID":728} "com.webos.service.memorymanager" does not have sufficient outbound permissions to communicate with "com.palm.webappmanager" (cmdline: /usr/sbin/memorymanager)

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Pending

 files/sysbus/com.webos.service.memorymanager.role.json.in | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/files/sysbus/com.webos.service.memorymanager.role.json.in b/files/sysbus/com.webos.service.memorymanager.role.json.in
index f6e379f..0e9ca30 100644
--- a/files/sysbus/com.webos.service.memorymanager.role.json.in
+++ b/files/sysbus/com.webos.service.memorymanager.role.json.in
@@ -8,8 +8,10 @@
             "service":"com.webos.service.memorymanager",
             "outbound": [
                 "com.webos.service.applicationmanager",
+                "com.webos.applicationManager",
                 "com.webos.notification",
-                "com.webos.service.account"
+                "com.webos.service.account",
+                "com.palm.webappmanager"
             ]
         }
     ]
